external/build/lib/dts-parse.js

```diff
@@ -348,7 +348,11 @@ class Transform {
     }
 
     if (node.type?.type === 'reference') {
-      if (chromeEventRefTypes.includes(node.type.name)) {
+      // Only upgrade node is not a workbox type
```

Comment:
I think think there's a mistake in this comment? s/is/if/

Also it might be worth adding a bit more context as to why the exception for workbox is made here. If someone else came to this code base it wouldn't be clear.